web/src/layout/Header/Logo.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled, { Theme, useTheme } from "styled-components";
+import KlerosCourtLogo from "svgs/header/kleros-court.svg";
+
+const arbitratorType = process.env.REACT_APP_ARBITRATOR_TYPE;
```

Comment:
Instead of getting the `arbitratorType` from the environment, I think it's better to use the function from `src/consts/index.tsx`.